pandas-stubs/core/series.pyi

```diff
@@ -1133,6 +1135,7 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
         convert_string: _bool = ...,
         convert_integer: _bool = ...,
         convert_boolean: _bool = ...,
+        dtype_backend: DtypeBackend | NoDefault = ...,
```

Comment:
remove `NoDefault` here .  When you make changes like this, check the docs and the pandas source to be sure of the right thing to use.

Reply:
Yes, i should have done that.